### PR TITLE
Issues #1 and #2 + some improvement in setting up the project for developers

### PR DESCRIPTION
**Pull request by m4nu56**
I added a .gitignore in public/data so that the folder is created because withtout it the `prep` was failing silently. 
Also added some logs to better identified when prep is failing. 

and did my best to solve issues #1 and #2 

I can't wait to chat with you about my changes :)

### CODE REVIEW

File: pages/details/[counter].tsx [hunk 1]
```diff
@@ -1,25 +1,25 @@
 import Head from 'next/head';
 import Link from 'next/link';
-import { GetStaticPaths, GetStaticProps } from 'next';
+import {GetStaticPaths, GetStaticProps} from 'next';
```
Comment (Tristramg) [inline pages/details/[counter].tsx]: I guess we have some linting issues here. I have no strong opinion on that matter, but it should be discussed in an other pull request, with a linter configuration and everything.

Can we skip for now reformating?

Comment (m4nu56) [reply pages/details/[counter].tsx]: Sure sorry about that, I'll update the PR

File: pages/details/[counter].tsx [hunk 2]
```diff
@@ -41,13 +41,15 @@ type Detail = {
...
-    <h3>{detail.name}</h3>
-    <p>Installé le {detail.date}</p>
-    <img src={detail.img} alt={`Image du compteur${detail.name}`}></img>
-    <SingleMarker coord={detail.coord} />
-  </div>
+const Detail = ({detail}: { detail: Detail }) => (
+    <div className="rounded-xl p-6 bg-white mb-4">
+        <h3>{detail.name}</h3>
+        <p>Installé le {detail.date}</p>
+        <a href={detail.img} target={'blank'}>
```
Comment (Tristramg) [inline pages/details/[counter].tsx]: No need to use interpolation here, just target="blank" will do

File: prepare.js [hunk 3]
```diff
@@ -160,8 +160,12 @@ async function save(data, metadata) {
     fs.writeFile(
       `public/data/${slugify(counter)}.json`,
       JSON.stringify(prepared),
-      () => {
-        console.log('Finished', counter);
+      (error) => {
+        if (error) {
+          console.error(`Error preparing ${counter} in public/data/${slugify(counter)}.json`, error)
```
Comment (Tristramg) [inline prepare.js:165]: :+1:

File: pages/details/[counter].tsx (hunk 1 above)
Comment (m4nu56) [reply pages/details/[counter].tsx]: After much fight against my IDE I finally found a way to revert my formatting changes..

**Review by Tristramg — CHANGES_REQUESTED** [review]

**Comment by Tristramg** [comment]
Thank you so much for taking that in charge! Some minor comments, but I’ll be glad to merge that!

**Comment by m4nu56** [comment]
Thank's for your review, it was a pleasure

**Pull request merged**